app/models/visit_metrics_entry.rb

```diff
@@ -2,7 +2,7 @@ class VisitMetricsEntry < ActiveRecord::Base
   validates_presence_of :visit_id, :nomis_id, :requested_at
   validates_inclusion_of :kind, in: %w{deferred}
```

Comment:
@dan-palmer because I didn't realise what it was for when I was shuffling migrations...